packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -0,0 +1,281 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+import Link from "@khanacademy/wonder-blocks-link";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelSmall} from "@khanacademy/wonder-blocks-typography";
+
+type BannerKind =
```

Comment:
Note: The banner types are in a different file in the mobile repo, but I kept them in the same file here to be more consistent with other web wonder blocks components. I can move them to a different file if that is preferable.